### PR TITLE
xmlcopyeditor: 1.2.1.3 -> 1.3.1.0

### PR DESCRIPTION
**Pull request by wegank**
###### Description of changes

Also migrated to wxGTK32 and added Darwin support.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/editors/xmlcopyeditor/default.nix [hunk 1]
```diff
@@ -1,28 +1,29 @@
-{ lib, stdenv, fetchurl, aspell, boost, expat, intltool, libxml2, libxslt, pcre, wxGTK, xercesc }:
+{ lib, stdenv, fetchurl, aspell, boost, expat, intltool, pkg-config, libxml2, libxslt, pcre, wxGTK32, xercesc, Cocoa }:
 
 stdenv.mkDerivation rec {
   pname = "xmlcopyeditor";
-  version = "1.2.1.3";
+  version = "1.3.0.0";
 
   src = fetchurl {
     name = "${pname}-${version}.tar.gz";
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/xmlcopyeditor/default.nix:8]:
```suggestion
```
this is redundant

File: pkgs/applications/editors/xmlcopyeditor/default.nix [hunk 2]
```diff
@@ -1,28 +1,29 @@
...
+  nativeBuildInputs = [ intltool pkg-config ];
+
+  buildInputs = [ aspell boost expat libxml2 libxslt pcre wxGTK32 xercesc ]
+    ++ lib.optionals stdenv.isDarwin [ Cocoa ];
 
   enableParallelBuilding = true;
 
   meta = with lib; {
     description = "A fast, free, validating XML editor";
     homepage = "http://xml-copy-editor.sourceforge.net/";
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/xmlcopyeditor/default.nix:24]:
```suggestion
    homepage = "https://xml-copy-editor.sourceforge.io/";
```

File: pkgs/applications/editors/xmlcopyeditor/default.nix (hunk 1 above)
Comment (wegank) [reply pkgs/applications/editors/xmlcopyeditor/default.nix]: Done.

File: pkgs/applications/editors/xmlcopyeditor/default.nix (hunk 2 above)
Comment (wegank) [reply pkgs/applications/editors/xmlcopyeditor/default.nix]: Done.

**Comment by wegank** [comment]
Bumped to 1.3.1.0.

**Pull request merged**